test/expected/ddl-13.out

```diff
@@ -487,10 +487,10 @@ SELECT * FROM alter_test WHERE time_us > '2017-05-20T10:00:01';
                                      QUERY PLAN                                      
 -------------------------------------------------------------------------------------
  Append
-   ->  Seq Scan on _hyper_9_4_chunk
-         Filter: (time_us > 'Sat May 20 10:00:01 2017'::timestamp without time zone)
    ->  Seq Scan on _hyper_9_5_chunk
          Filter: (time_us > 'Sat May 20 10:00:01 2017'::timestamp without time zone)
+   ->  Seq Scan on _hyper_9_4_chunk
```

Comment:
I see a lot of changes like this where the plan branches change places. I was wondering what's the reason?

Reply:
Yeah, so the order of the children in an append is governed by the order in which chunks are discovered during the parent expansion. Since we change how we scan for the chunks that should be included, the order also changed. We could sort the chunks according to, e.g., relation ID or similar to always have the same order, but that is an unnecessary sorting step that will only slow things down.